presto-postgresql/src/main/java/io/prestosql/plugin/postgresql/PostgreSqlClient.java

```diff
@@ -659,7 +660,8 @@ private static JdbcTypeHandle getArrayElementTypeHandle(Connection connection, J
                     Optional.of(typeInfo.getPGType(pgElementOid)),
                     arrayTypeHandle.getColumnSize(),
                     arrayTypeHandle.getDecimalDigits(),
-                    arrayTypeHandle.getArrayDimensions());
+                    arrayTypeHandle.getArrayDimensions(),
+                    arrayTypeHandle.getCaseSensitivity());
```

Comment:
can array be case sensitive?

Reply:
I think its content could be case sensitive

Reply:
i think we are not handling this correctly in any way.
rather, i'd have here `verify(arrayTypeHandle.getCaseSensitivity().isEmpty())`.